code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -844,3 +844,24 @@
 	message_admins("[ADMIN_LOOKUPFLW(obsessed)] has been made Obsessed by the midround ruleset.")
 	log_game("[key_name(obsessed)] was made Obsessed by the midround ruleset.")
 	return TRUE
+
+/// Space Changeling ruleset
+/datum/dynamic_ruleset/midround/from_ghosts/changeling_midround
+	name = "Space Changeling"
+	midround_ruleset_style = MIDROUND_RULESET_STYLE_LIGHT
```

Comment:
We're definitely going to have to keep a very close eye on this because changelings can be wicked deadly